### ISSUE TITLE
Accessibility: missing font size settings

### ISSUE THREAD

**Issue by Akeel1994** [mastodon/mastodon-ios#387]
Hey, I have a low vision and the font size in your Android and iOS app is just too small. So please add an option to customise the font size e.g. from 10-40. So everyone could choose the right size. Thank you!

**Comment by MainasuK** [comment]
Hello, @Akeel1994 

The app supports system Dynamic Type feature. The font size should get right size from your preferred system font size. If that size still too small. Please try to set the font size for specific app. There is a [guide](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center) describes how to do it.

**Comment by MainasuK** [comment]
> Yeah but everything is just too small. The guide won't help for the Android version.
> So an extra font size setting in the app would be great.
> The dynamic font size in the current state is just not good enough for people with a low vision like me

Oh, I see. 

If your turn on the large text on iOS. There are five extra accessibility scales avaiable. No too many avaiable ranges to control the font. So I needs to figure out which font size meets your needs. 

Here are some example. Please open the image on your phone then check the post body font size.  

| Dynamic Type<br>(max scale)  | 60pt | 80pt | 100pt |
| ----------- | ----------- | ----------- | ----------- | 
| <img width = 100 src="https://user-images.githubusercontent.com/7940186/165649510-5816d9ce-44d5-4629-a070-c9e076b82699.png">      | <img width=100 src="https://user-images.githubusercontent.com/7940186/165649939-993b43e0-c5a3-4eac-a069-0d28fc5b7912.png"> | <img width=100 src="https://user-images.githubusercontent.com/7940186/165650158-56331b0d-52fc-450d-aeda-724b3fbd5673.png"> | <img width=100 src="https://user-images.githubusercontent.com/7940186/165650486-75792b3d-5b32-4a27-8d94-eaee38b43aed.png" >

**Comment by Akeel1994** [comment]
I don't have an iPhone so I can't tell you with the screenshot you posted. 
I installed the app one the iPad Pro 11 and the Pixel 6 Pro. 

Here is an example from the Pixel. You can see my twitter app in comparison.  On the twitter app which I'm using I can read the text. The app allows me to set up the font size. But on the Mastodon App it's really hard to read and there is no option to customize the size.

**Comment by Akeel1994** [comment]
> Hello, @Akeel1994 
> 
> 
> 
> The app supports system Dynamic Type feature. The font size should get right size from your preferred system font size. If that size still too small. Please try to set the font size for specific app. There is a [guide](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center) describes how to do it. 

Okay I tried this on the iPad and now it's fine. So only the Android app has the problem now.

**Comment by MainasuK** [comment]
<img width="300" alt="image" src="https://user-images.githubusercontent.com/7940186/165654606-2f836aad-f781-46de-ad6f-6650ccd58126.png">

I'm trying compare it side by side. The iOS Dynamic Type font size should meets your needs. That's looks larger than the Twitter client. To using the max scale Dynamic Type. Please follow the [guide](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center) and there is maximum 310% scale radio avaiable. 

For Android device. There is a related issue about it. Also, you can create a new one here: https://github.com/mastodon/mastodon-android/issues

**Comment by MainasuK** [comment]
> > Hello, @Akeel1994
> > The app supports system Dynamic Type feature. The font size should get right size from your preferred system font size. If that size still too small. Please try to set the font size for specific app. There is a [guide](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center) describes how to do it.
> 
> Okay I tried this on the iPad and now it's fine. So only the Android app has the problem now.

Glad to see it. 

There is a plan to make an in-app control. But the poor systesm API break something. I needs some research in future.

**Comment by sammcj** [comment]
Personally I'd really like to be able to set the font size a lot smaller, and if possible select a font that I find the most legible.

**Comment by DirkR** [comment]
I just installed the app on a mac with ARM CPU and the app is running smooth. But there is no possibility to change the font size anywhere. It would be very helpful to be able to manually increase the font size.

**Comment by jayholler** [comment]
+1 adjusting text size in macOS doesn't seem possible on a per-app basis.

**Comment by likethegoddess** [comment]
> Hello, @Akeel1994
> 
> The app supports system Dynamic Type feature. The font size should get right size from your preferred system font size. If that size still too small. Please try to set the font size for specific app. There is a [guide](https://www.theverge.com/22580423/ios-15-iphone-text-size-how-to-control-center) describes how to do it.

This Dynamic Type feature is not changing type size for me on iPhone 12 mini, latest iOS.

**Comment by j-f1** [comment]
What text isn’t changing size? Posts should change instantly, but some other text may require a restart of the app to show up.

**Comment by likethegoddess** [comment]
Hmm, seems to be working now.

**Comment by tofraley** [comment]
I'm also using the iOS app on a m1 MacBook Air.  I don't see any way to adjust the font, and the iPhone guides do not help.  Not even sure why the default size is so small.  I don't see an option to even set that.  But it would be great to be able to adjust it 👍

**Comment by smxi** [comment]
Just a nudge, a friend tried the android app just now and can't use it because she can't adjust the font size and it's too small for her to read.
